src/dfx/src/lib/sns/create_config.rs

```diff
@@ -1,40 +1,21 @@
-use anyhow::{anyhow, Context};
+//! Code for executing `dfx sns config create`
 use fn_error_context::context;
+use std::ffi::OsString;
 use std::path::Path;
-use std::process::{self, Command};
 
 use crate::lib::error::DfxResult;
+use crate::lib::sns::sns_cli::call_sns_cli;
 use crate::Environment;
 
+/// Executes `dfx sns config create`
 #[context("Failed to create sns config at {}.", path.display())]
 pub fn create_config(env: &dyn Environment, path: &Path) -> DfxResult {
-    let cli_name = "sns";
-    let sns_cli = env
-        .get_cache()
-        .get_binary_command_path(cli_name)
-        .with_context(|| format!("Could not find bundled binary '{cli_name}'."))?;
-    let mut command = Command::new(sns_cli);
-    command
-        .arg("init-config-file")
-        .arg("--init-config-file-path")
-        .arg(path)
-        .arg("new");
-    command
-        .stdin(process::Stdio::null())
-        .output()
-        .map_err(anyhow::Error::from)
-        .and_then(|output| {
-            if output.status.success() {
-                Ok(())
-            } else {
-                Err(anyhow!(
-                    "Failed to create an SNS configuration.\n{:?} {:?}\nStdout:\n{:?}\n\nStderr:\n{:?}",
```

Comment:
Note: The context message on the function provides the important contextual information, and the library function provides stdout and stderr if it fails.